clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -73,6 +75,7 @@ public class RecordAccumulator {
     private final CompressionType compression;
     private final int lingerMs;
     private final long retryBackoffMs;
+    private final ExponentialBackoff retryBackoff;
```

Comment:
There is still one reference of retryBackoffMs in batchReady(). Should that be changed with exponential backoff?